src/sentry/static/sentry/app/views/organizationIntegrations/pluginDetailedView.tsx

```diff
@@ -49,7 +50,9 @@ class PluginDetailedView extends AbstractIntegrationDetailedView<
   }
 
   get integrationName() {
-    return this.plugin.name;
+    const isLegacy = legacyIds.includes(this.plugin.id);
+    const displayName = `${this.plugin.name} ${!!isLegacy ? '(Legacy)' : ''}`;
```

Comment:
`.includes` already returns a Boolean so the `!!` isn't needed

Reply:
Also it looks like this would always result in an extra space

Reply:
@leeandher I don't think would matter because it's in the title. There  is no other text in that `div`.